mods/general_content/species/lizard.dm

```diff
@@ -0,0 +1,103 @@
+/datum/species/unathi
+	name = SPECIES_UNATHI
+	name_plural = SPECIES_UNATHI
+	icon_template = 'icons/mob/human_races/species/template_tall.dmi'
+	icobase = 'icons/unathi/body.dmi'
+	deform = 'icons/unathi/deformed_body.dmi'
+	husk_icon = 'icons/unathi/husk.dmi'
+	preview_icon = 'icons/unathi/preview.dmi'
+	bandages_icon = 'icons/mob/bandage.dmi'
+	tail = "sogtail"
+	tail_animation = 'icons/unathi/unathi_tail.dmi'
+	limb_blend = ICON_MULTIPLY
+	tail_blend = ICON_MULTIPLY
+	hidden_from_codex = FALSE
+
+	//unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/tail, /datum/unarmed_attack/claws, /datum/unarmed_attack/punch, /datum/unarmed_attack/bite/sharp)
+	primitive_form = "Stok"
+	darksight_range = 4
+	darksight_tint = DARKTINT_MODERATE
+	gluttonous = GLUT_TINY
+	strength = STR_HIGH
+	breath_pressure = 18
+	slowdown = 0.5
+	brute_mod = 0.8
+	flash_mod = 1.2
+	blood_volume = 800
+
+	health_hud_intensity = 2
+	hunger_factor = DEFAULT_HUNGER_FACTOR * 2
+
+	min_age = 18
+	max_age = 280
+
+	body_temperature = null //Makes their body temperature the same as environment, cold blooded
+
+	description = "A heavily reptillian species, Unathi (or 'Sinta as they call themselves) hail from the \
+	Uuosa-Eso system, which roughly translates to 'burning mother'.<br/><br/>Coming from a harsh, radioactive \
+	desert planet, they mostly hold ideals of honesty, virtue, and bravery above all else, \
+	frequently even their own lives. They prefer warmer temperatures than most species and \
+	their native tongue is a heavy hissing laungage called Sinta'Unathi."
+
+	cold_level_1 = 280 //Default 260 - Lower is better
+	cold_level_2 = 220 //Default 200
+	cold_level_3 = 130 //Default 120
+
+	heat_level_1 = 420 //Default 360 - Higher is better
+	heat_level_2 = 480 //Default 400
+	heat_level_3 = 1100 //Default 1000
+
+	spawn_flags = SPECIES_CAN_JOIN | SPECIES_NO_ROBOTIC_INTERNAL_ORGANS
+	appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
+
+	flesh_color = "#34af10"
+
+	base_color = "#066000"
+	blood_color = "#f24b2e"
+	organs_icon = 'icons/unathi/organs.dmi'
+
+	move_trail = /obj/effect/decal/cleanable/blood/tracks/claw
+
+	heat_discomfort_level = 320
+	heat_discomfort_strings = list(
+		"You feel soothingly warm.",
+		"You feel the heat sink into your bones.",
+		"You feel warm enough to take a nap."
+		)
+
+	cold_discomfort_level = 290
+	cold_discomfort_strings = list(
+		"You feel chilly.",
+		"You feel sluggish and cold.",
+		"Your scales bristle against the cold."
+		)
+	breathing_sound = 'sound/lizard.ogg'
+
+	/*base_auras = list(
```

Comment:
You should be able to straight up copypaste the aura definition from Bay to Neb and have this work.

Reply:
The Unathi loredev requested I have this disabled for the meantime pending lore changes.